stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -638,6 +648,30 @@ public static void main(String[] args) {
     cli(args, Starter.class);
   }
 
+  private class BundleFailureListener implements FrameworkListener {
+
+    @Override
+    public void frameworkEvent(FrameworkEvent event) {
+      Throwable throwable = event.getThrowable();
+      if (throwable != null) {
+        // We rely on the exception class name here because there is no easy way to share
+        // classes between the Starter and bundles, and this use case does not seem worth
+        // adding a new OSGi artifact.
+        if (throwable.getClass().getSimpleName().equals("ServiceStartException")) {
+          startError.set(true);
+
+          System.out.printf(
+              "Detected service startup failure in bundle %s: %s",
+              event.getBundle().getSymbolicName(), throwable);
+          System.out.println();
```

Comment:
Not a big deal. According to here: https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html you can use `%n` for a platform-specific newline.

Reply:
TIL, but I'd like to avoid new changes to save build time... if the build fails for some reason, I'll use `%n`.

Reply:
fixed